src/coop/rchain/lang/Rho.bnf

```diff
@@ -173,7 +178,7 @@ QVal ::= TRUE
 
 // Quantity
 Quantity_ ::= ("-"|"+") Quantity_
-  | "(" [Proc] ")"
+  | ParenthesizedProc
   | "[" [Proc] "]"
   | "{" [Proc] "}"
```

Comment:
We missed this the last time. It is also `ProcBlock`.

Reply:
Yeah, you're right.